docs/io.cozy.calendar.md

```diff
@@ -0,0 +1,54 @@
+[Table of contents](README.md#table-of-contents)
+    
+# Cozy Calendar doctype
+
+## `io.cozy.calendar`
+
+The `io.cozy.calendar` doctype is loosely based on the [iCalendar RFC](https://datatracker.ietf.org/doc/html/rfc5545), in that some of the attributes have been renamed for clarity. The attributes with a `?` are optional.
+
+- `start?` : {date} (example: `"1959-05-15"`)
+- `end?` : {date} (example: `"1959-05-15"`)
+- `label?` : {string}
+- `description?` : {string}
+- `location?` : {string}
+- `organiser?` : {string}
+- `status?` : {string} `CONFIRMED|CANCELLED`
+- `attendee?` : {object} Array of string with event attendees
+
+### Example
+```json
+{
+  "_id": "62e5d66d6e11d19992b7efce794263f0",
+  "start": "20170422T010000",
+  "end": "20170422T010000",
+  "label": "history-geography"
+  "location": "B209 BIS",
+  "organizer": "Mme. Dubois",
+  "status": "CONFIRMED",
+  "attendee": ["TG3", "1G4"],
+  "description": "Apportez vos manuels !"
+}
+```
+
+## `io.cozy.calendar.event`
```

Comment:
In general, we prefer plural for doctype names, like `io.cozy.calendars.events`.

Reply:
I'm waiting for these pull requests to change that in this PR
https://github.com/konnectors/pronote/pull/12
https://github.com/cozy/cozy-papillon-app/pull/21